installer/mod.ts

```diff
@@ -1,145 +1,16 @@
-    case "--allow-write":
-      return Permission.Write;
-    case "--allow-net":
-      return Permission.Net;
-    case "--allow-env":
-      return Permission.Env;
-    case "--allow-run":
-      return Permission.Run;
-    case "--allow-all":
-      return Permission.All;
-    case "-A":
-      return Permission.All;
-  }
-}
-
-function getFlagFromPermission(perm: Permission): string {
-  switch (perm) {
-    case Permission.Read:
-      return "--allow-read";
-    case Permission.Write:
-      return "--allow-write";
-    case Permission.Net:
-      return "--allow-net";
-    case Permission.Env:
-      return "--allow-env";
-    case Permission.Run:
-      return "--allow-run";
-    case Permission.All:
-      return "--allow-all";
-  }
-  return "";
-}
-
-async function readCharacter(): Promise<string> {
-  const byteArray = new Uint8Array(1024);
-  await stdin.read(byteArray);
-  const line = decoder.decode(byteArray);
-  return line[0];
-}
-
-async function yesNoPrompt(message: string): Promise<boolean> {
-  console.log(`${message} [yN]`);
-  const input = await readCharacter();
-  console.log();
-  return input === "y" || input === "Y";
-}
-
-function createDirIfNotExists(path: string): void {
-  try {
-    readDirSync(path);
-  } catch (e) {
-    mkdirSync(path, true);
-  }
-}
-
-function checkIfExistsInPath(filePath: string): boolean {
-  // In Windows's Powershell $PATH not exist, so use $Path instead.
-  // $HOMEDRIVE is only used on Windows.
-  const { PATH, Path, HOMEDRIVE } = env();
-
-  let envPath = (PATH as string) || (Path as string) || "";
-
-  const paths = envPath.split(isWindows ? ";" : ":");
-
-  let fileAbsolutePath = filePath;
-
-  for (const p of paths) {
-    const pathInEnv = path.normalize(p);
-    // On Windows paths from env contain drive letter.
-    // (eg. C:\Users\username\.deno\bin)
-    // But in the path of Deno, there is no drive letter.
-    // (eg \Users\username\.deno\bin)
-    if (isWindows) {
-      if (driverLetterReg.test(pathInEnv)) {
-        fileAbsolutePath = HOMEDRIVE + "\\" + fileAbsolutePath;
-      }
-    }
-    if (pathInEnv === fileAbsolutePath) {
-      return true;
-    }
-    fileAbsolutePath = filePath;
-  }
-
-  return false;
-}
-
-function getInstallerDir(): string {
-  // In Windows's Powershell $HOME environmental variable maybe null
-  // if so use $HOMEPATH instead.
-  let { HOME, HOMEPATH } = env();
-
-  const HOME_PATH = HOME || HOMEPATH;
-
-  if (!HOME_PATH) {
-    throw new Error("$HOME is not defined.");
-  }
-
-  return path.join(HOME_PATH, ".deno", "bin");
-}
+const { args, exit } = Deno;
+import { install } from "./install.ts";
+import { uninstall } from "./uninstall.ts";
 
 function showHelp(): void {
   console.log(`deno installer
   Install remote or local script as executables.
 
 USAGE:
-  deno https://deno.land/std/installer/mod.ts EXE_NAME SCRIPT_URL [FLAGS...]
+  deno https://deno.land/std/installer/mod.ts install EXE_NAME SCRIPT_URL [FLAGS...]
```

Comment:
This will bring a breaking change.

Should we need to set the default value to `install` to void BREAKING CHANGE?

Reply:
and don't forget the `readme.md` 😄

Reply:
> This will bring a breaking change.
> 
> Should we need to set the default value to `install` to void BREAKING CHANGE?

In main repo `INSTALLER_URL` is tagged so it won't break.